BWEB.h

```diff
@@ -3,7 +3,7 @@
 #include <set>
 
 #include <BWAPI.h>
-#include "..\BWEM\bwem.h"
+#include <BWEM\bwem.h>

```

Comment:
shouldn't this be just `#include <BWEM.h>` for full compatibility?

Reply:
Probably you are right, I may be confused by the BWAPI include structure. I never seen the BWEM code, but I suspect that I may be wrong with this change. Still decide to make it like this, to collect feedback.

Reply:
I just don't think we should force anyone to have their code in a directory called `BWEM`. This could for example be an issue with the (hopefully) upcoming BWEM-community.